app/views/pages/_dashboard_assets_overview.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="border-b border-gray-200 bg-white p-4 my-2 rounded">
+  <h2 class="text-sm text-center text-gray-500"><%= t('.title') %></h2>
+
+  <% if @account_groups.any? %>
+    <div class="my-4">
+      <h3 class="text-2xl font-bold text-center text-gray-800"><%= @account_groups.sum(&:total_value).format(no_cents: true) %></h3>
+    </div>
+
+    <div class="flex -mx-0.5 my-6">
+      <% @account_groups.each do |account_group| %>
+        <div class="block mx-0.5 h-2.5 rounded bg-<%= account_group.param %>" style="flex-basis: <%= account_group.percentage_held %>%"></div>
+      <% end %>
+    </div>
+
+    <div class="flex justify-center mt-8">
+      <% @account_groups.each do |account_group| %>
+        <div class="flex items-center mx-4">
+          <span class="inline-block rounded-full w-3 h-3 bg-<%= account_group.param %>"></span>
```

Comment:
@robzolkos @JoshAntBrown 
This is the dynamic class definition I'm talking about.

If we extend tailwind confit with a new color `account-credit`.

Without actually writing `bg-account-credit` in template, tailwind JIT can't find it and will purge the class from the build.

Reply:
You can use tailwinds [safelist](https://tailwindcss.com/docs/content-configuration#safelisting-classes) feature for this.   I'm just wondering if its premature however as I don't think we're using these colors anywhere else yet (and a solution may become clearer when we do?)

Reply:
That makes sense, but yes it's quite premature now to add IMO. But let me know if you'd prefer we go that route.